### PR TITLE
feature/global-styles

### PR DESCRIPTION
**Pull request by billyxs**

### CODE REVIEW

**Comment by seesleestak** [comment]
Nice, was wondering if we wanted to do something like this

**Comment by billyxs** [comment]
Made this to start a conversation. I'm sure there are opinions out there.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ import Container from '@hixme-ui/container'
 import Title from '@hixme-ui/title'
 
 const AppContainer = () => (
-  <ThemeProvider>
+  <ThemeProvider withGlobal>
```
Comment (seesleestak) [inline README.md]: Cool stuff, I dig this.

Comment (billyxs) [reply README.md]: thought this would be a good way to opt in and update explicitly

**Pull request merged**